apps/balancer-tools/src/app/apr/pool/(components)/HistoricalAPRChart.tsx

```diff
@@ -0,0 +1,56 @@
+import { blueDark } from "@radix-ui/colors";
+
+import { trimTrailingValues } from "#/lib/utils";
+
+import { calculateAPRForPool } from "../../(utils)/calculateRoundAPR";
+import { Round } from "../../(utils)/rounds";
+import HistoricalAPRPlot from "./HistoricalAPRPlot";
+
+export default async function HistoricalAPRChart({
+  roundId,
+  poolId,
+}: {
+  roundId: string;
+  poolId: string;
+}) {
+  const HOVERTEMPLATE = "%{x}<br />%{y:.2f}% APR<extra></extra>";
+  const LAST_ROUND_ID = parseInt(Round.getAllRounds()[0].value);
```

Comment:
would move it out of the component if it's not dependent on the local logic here